language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSPackageLoader.java

```diff
@@ -124,24 +125,40 @@ public List<PackageInfo> getDistributionRepoPackages() {
      *
      * @return {@link List} packages
      */
-    public List<PackageInfo> getAllVisiblePackages(DocumentServiceContext ctx) {
-        Map<String, PackageInfo> packagesList = new HashMap<>();
+    public List<ModuleInfo> getAllVisiblePackages(DocumentServiceContext ctx) {
+        Map<String, ModuleInfo> packagesList = new HashMap<>();
         this.getDistributionRepoPackages().forEach(packageInfo ->
                 packagesList.put(packageInfo.packageIdentifier(), packageInfo));
-        List<PackageInfo> repoPackages = this.getPackagesFromBallerinaUserHome(ctx);
+        List<ModuleInfo> repoPackages = this.getPackagesFromBallerinaUserHome(ctx);
         repoPackages.stream().filter(packageInfo -> !packagesList.containsKey(packageInfo.packageIdentifier()))
                 .forEach(packageInfo -> packagesList.put(packageInfo.packageIdentifier(), packageInfo));
+        Package currentPackage = ctx.workspace().project(ctx.filePath()).get().currentPackage();
+        currentPackage.modules().forEach(module -> {
+            Package packageInstance = module.packageInstance();
+            ModuleInfo moduleInfo = new ModuleInfo(PackageOrg.from(""), packageInstance.packageName(),
```

Comment:
Why are we using `PackageOrg.from("")` instead of `packageInstance.packageOrg()`?

Reply:
Here, we are creating a new empty packageOrg with the intention of not including it in the module import.